src/uwtools/cli.py

```diff
@@ -43,24 +43,19 @@ def main() -> None:
     # are known, then dispatch to the [sub]mode handler.
 
     setup_logging(quiet=True)
-    try:
-        args, checks = _parse_args(sys.argv[1:])
-        for check in checks[args[STR.mode]][args[STR.action]]:
-            check(args)
-        setup_logging(quiet=args[STR.quiet], verbose=args[STR.verbose])
-        log.debug("Command: %s %s", Path(sys.argv[0]).name, " ".join(sys.argv[1:]))
-        modes = {
-            STR.config: _dispatch_config,
-            STR.fv3: _dispatch_fv3,
-            STR.rocoto: _dispatch_rocoto,
-            STR.sfcclimogen: _dispatch_sfc_climo_gen,
-            STR.template: _dispatch_template,
-        }
-        sys.exit(0 if modes[args[STR.mode]](args) else 1)
-    except Exception as e:  # pylint: disable=broad-exception-caught
-        if _switch(STR.debug) in sys.argv:
-            log.exception(str(e))
-        _abort(str(e))
+    args, checks = _parse_args(sys.argv[1:])
+    for check in checks[args[STR.mode]][args[STR.action]]:
+        check(args)
+    setup_logging(quiet=args[STR.quiet], verbose=args[STR.verbose])
+    log.debug("Command: %s %s", Path(sys.argv[0]).name, " ".join(sys.argv[1:]))
+    modes = {
+        STR.config: _dispatch_config,
+        STR.fv3: _dispatch_fv3,
+        STR.rocoto: _dispatch_rocoto,
+        STR.sfcclimogen: _dispatch_sfc_climo_gen,
+        STR.template: _dispatch_template,
+    }
+    sys.exit(0 if modes[args[STR.mode]](args) else 1)
```

Comment:
An unhandled exception will now propagate all the way up. If we find there are cases we can deal with better than this default behavior, we can catch and handle.